src/darker/command_line.py

```diff
@@ -30,6 +30,12 @@ def parse_command_line(argv: List[str]) -> Namespace:
         help="Path(s) to the Python source file(s) to reformat",
         metavar="PATH",
     )
+    parser.add_argument(
+        "-r",
+        "--revision",
+        default="HEAD",
+        help="Git revision against which to compare the working tree",
```

Comment:
To match your suggestion in the README:
```suggestion
        help=(
            "Git revision against which to compare the working tree. Tags, branch"
            " names, commit hashes, and other expressions like HEAD~5 work here."
        ),
```